pyro/primitives.py

```diff
@@ -238,17 +229,17 @@ class iarange(object):
     See `SVI Part II <http://pyro.ai/examples/svi_part_ii.html>`_ for an
     extended discussion.
     """
-    def __init__(self, name, size=None, subsample_size=None, subsample=None, dim=None, use_cuda=None):
+    def __init__(self, name, size=None, subsample_size=None, subsample=None, dim=None, device=None):
```

Comment:
I think we should keep a backwards-compatible `use_cuda` around for the remainder of the 0.2.x releases, and then drop it in the 0.3 release. I'm planning on a short-lived 0.2.2 release following the next PyTorch release, then a breaking 0.3 release 3-6 weeks later when we have a tech report. Sound reasonable?

Reply:
And thanks for cleaning this up!

Reply:
Sounds good; it might be best to add it to dev.